packages/ember-testing/tests/helpers_test.js

```diff
@@ -512,6 +512,33 @@ test("`fillIn` takes context into consideration", function() {
   });
 });
 
+test("`fillIn` focuses on the element", function() {
+  expect(2);
+  var fillIn, find, visit, andThen;
+
+  App.IndexView = EmberView.extend({
+    template: compile('<div id="parent">{{input type="text" id="first"}}</div>'),
+    didInsertElement: function() {
+      this.$('#first').on('focusin', function() {
```

Comment:
Can you instead just bind the `focus-in` action?  Something like http://emberjs.jsbin.com/rwjblue/193/edit.

Then you can assert that the action was called.